### PR TITLE
Seed data

### PR DESCRIPTION
**Pull request by ttreit**
Created script to seed data and added script to package.json and instructions for using the script in README.

Manually tested seeding script - have not yet installed testing frameworks.

### CODE REVIEW

**Comment by rvrita** [comment]
Overall awesome job, you nailed it with the seeding script. I have a few suggestions but your code is very neatly organized and great job adding a readme and a process.env at the beginning ;)

In you shipping-seeder.js file you are running your script syncronously but you have async code right before that. It would be better practice to add your script inside the .then() block even if you just logging something out.

Instead of constructing and saving instances of the model, consider using the create function. It can save multiple documents to the database: 

after your `mongoose.connect('mongodb://localhost:27017/fecShipping', { useNewUrlParser: true })` you can use 

```
 Product.create(products) 
    .then(() => closeConnection());
```
 
where products is the array exported from mongo-mockdata/all.json 

 Using this you would not need the counter variable either, it would disconnect after the last instance were created, and the seeding-shipper.js would be only a few lines long. 

Happy to chat about it if you have questions, but here is the documentation about Model.create():
 https://mongoosejs.com/docs/api.html#model_Model.create

**Comment by ttreit** [comment]
Really great observations - because this works as is I'm going to go ahead and merge with master and make a ticket to consider these refactors if time allows. I don't know how I missed model.create!

File: seed/shipping-seeder.js [hunk 1]
```diff
@@ -0,0 +1,1039 @@
+//requirements
+const mongoose = require('mongoose');
+const Product = require('../models/shipping'); //schema for this collection
+
+//List of items to add to my dev and test databases - product_id 1-100 match product ids from other team member's dbs. The fake data may not match.
+const products = [
```
Comment (nickfreemandesign) [inline seed/shipping-seeder.js:6]: @ttreit  please refactor this to not contain static data but auto-generated from a pipeline.

Comment (ttreit) [reply seed/shipping-seeder.js:6]: I'm not sure I understand what you mean.

Comment (nickfreemandesign) [reply seed/shipping-seeder.js:6]: i mean to say that your seed file is simply static data with a single insert, and it should be a data pipeline composed of functions which generate this data and then insert into your database

**Pull request merged**